buddy_init/base_skeleton/src/config.py

```diff
@@ -0,0 +1,28 @@
+import math
+import os
+
+import jax.experimental.optimizers
+
+import experiment_buddy
+
+initial_lr = .0001
+
+decay_steps = 500000
+num_hidden = 1024
+decay_factor = .5
+
+batch_size = 128
+momentum_mass = 0.99
+weight_norm = 0.00
+
+num_epochs = 1000
+
+experiment_buddy.register(locals())
+
+################################################################
+# Derivative parameters
+################################################################
+learning_rate = jax.experimental.optimizers.inverse_time_decay(initial_lr, decay_steps, decay_factor, staircase=True)
+eval_every = math.ceil(num_epochs / 1000)
+
+tensorboard = experiment_buddy.deploy(host=os.environ.get('BUDDY_HOST', ""), sweep_yaml="")
```

Comment:
where do we set `export BUDDY_HOST`?

Reply:
At the moment is duty of the user

Reply:
lets' make it explicit in the readme of the example then

Reply:
At the moment the example doesn't have a readme, on my way

Reply:
no need for a full readme just +1 line on how to activate the cluster